call-notes/call_7.md

```diff
@@ -0,0 +1,212 @@
+# Core Community Call 7 Notes
+
+Meeting Date/Time: Friday, June 16th, 2023 19:00 UTC
+
+Meeting Duration: 31 mins
+
+[Video of the meeting](https://www.youtube.com/watch?v=9m_M8zEw1cE)
+
+## Decisions
+
+- There may be need to split the implementation of [SIMD 0052](https://github.com/solana-foundation/solana-improvement-documents/pull/52) into two or more different SIMDs to address different methods of implementation and concerns surrounding the SIMD 0052.
+
+## Speakers
+
+1. [Jacob Creech](https://github.com/jacobcreech)
+2. [Anoushk | Tinydancer](https://github.com/anoushk1234)
+3. [Richie Patel](https://github.com/ripatel-jump) - (Refered to as Harsh by Jacob Creech)
```

Comment:
This was actually Richard Patel, I was asking if Hersh was also on the call at one point. He was not.

Reply:
> This was actually Richard Patel, I was asking if Hersh was also on the call at one point. He was not.

@jacobcreech I have pushed an update with clarification between the two speakers to help remove the confusion that may arise for note readers.

Thanks for the clarification!